apps/docs/content/guides/platform/backups.mdx

```diff
@@ -233,3 +233,23 @@ Common examples of check constraints that can result in such failures are:
 
 - validating against the current time, e.g. that the row being inserted references a future event
 - validating the contents of a row against the contents of another table
+
+#### Views that reference themselves
+
+Views that directly or indirectly reference themselves can cause **cyclic dependency errors** during a logical backup restore. These views are also **invalid and unusable** , any query against them will result in an error at runtime.
+
+**Example:**
+
+```
+-- Direct self-reference
+CREATE VIEW my_view AS
+  SELECT * FROM my_view;
+
+-- Indirect circular reference
+CREATE VIEW v1 AS SELECT * FROM v2;
+CREATE VIEW v2 AS SELECT * FROM v1;
+```
+
+-- Drop the offending view.
```

Comment:
stray comment? Can make it a full sentence e.g. `Drop the offending view from your database, or delete them from the logical backup to make it restorable.`